static/js/rendered_markdown.js

```diff
@@ -166,11 +166,12 @@ export const update_elements = (content) => {
         if (isValid(timestamp)) {
             const text = $(this).text();
             const rendered_time = timerender.render_markdown_timestamp(timestamp, text);
+            const rendered_text = rendered_time.text + ` ${timerender.get_abbreviated_timezone()}`;
```

Comment:
This logic can’t be right, since the abbreviated timezone needs to vary with daylight saving time depending on the time being rendered, *not* on the current time. For example, in America/Los_Angeles, 24 hours after March 13, 2021 1:00 PM PST was March 14, 2021 2:00 PM PDT.

Reply:
Oh Thanks, I looked into it and seems like there is an easy way using <a href = 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat'>formatToParts()</a>.

I implemented as follows: 

```javascript
  new Intl.DateTimeFormat([], {timeZoneName: "short", timeZone: "America/Los_Angeles"}).formatToParts(time);
```
And by altering the `time` value across `March 13, 2021` and `March 14, 2021`, it produces `GMT-7` and `GMT-8` as `timeZoneName`.

Reply:
@aryanshridhar it looks like you haven't updated this PR since this comment?

Reply:
Yup, Anders reviewed it and its still unclear as our main intention is to get the <code>abbreviated timezone</code> and not the GMT offset.